types/ramda/test/indexBy-tests.ts

```diff
@@ -0,0 +1,15 @@
+import * as R from 'ramda';
+
+() => {
+  interface Book {
```

Comment:
Possibly, though DefinitelyTyped restricts what files you can have where, so it may not like it. Still, I think it would allow something like:

```
- test/
  - helpers/
    - models.ts
  - bar-tests.ts
  - baz-tests.ts
```

I'd just want to avoid having a `test/test-helpers.ts` file that would get lost in the sea of files under `test/`.

At any rate, I'd like to hold off on it for this PR and have it be part of the wider test improvements that will come after the `ramda-tests.ts` split is complete.